packages/block-editor/src/components/block-list/breadcrumb.native.js

```diff
@@ -91,13 +95,14 @@ export default compose( [
 		}
 		const rootBlockName = getBlockName( rootClientId );
 		const rootBlockType = getBlockType( rootBlockName );
-		const rootBlockIcon = rootBlockType.icon;
+		const rootBlockIcon = rootBlockType ? rootBlockType.icon : {};
 
 		return {
 			clientId,
 			blockIcon,
 			rootClientId,
 			rootBlockIcon,
+			isRTL: I18nManager.isRTL,
```

Comment:
Why you added it here instead of using it directly? You could even move this logic to the `SubdirectorSVG`. WDYT?

Reply:
I refactored to use it directly and keep login here according to what we discussed